src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import duke.main.Parser;
+import duke.task.TaskList;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void parseCommand_unknownCommand() {
+        TaskList dummyTaskList = new TaskList();
+        assertEquals(("☹ OOPS!!! I'm sorry, but I don't know what that means :-("),
+            Parser.parseCommand("hello", dummyTaskList));
```

Comment:
I think you can set ur IDE such that indentation for wrapped lines is 8 spaces. If you are using intellij, under code style->Java->Tabs and Indents, you can set continuation indent = 8

Reply:
Resolving this as per my comment on the first instance of _"checkstyle.xml Validation"_ being pointed out! Thanks for being thorough, nonetheless!